tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Subscriber/truncateUsedCSSHandler.php

```diff
@@ -25,20 +26,18 @@ class Test_TruncateUsedCSSHandler extends TestCase {
 	private $database;
 	private $usedCSS;
 	private $subscriber;
-	private $options_api;
-	private $homepage_preloader;
+	private $queue;
 
 	protected $path_to_test_data = '/inc/Engine/Optimization/RUCSS/Admin/Subscriber/truncateUsedCSSHandler.php';
 
 	public function setUp() : void {
 		parent::setUp();
 
-		$this->settings    = Mockery::mock( Settings::class );
-		$this->database    = Mockery::mock( Database::class );
-		$this->usedCSS     = Mockery::mock( UsedCSS::class );
-		$this->options_api = Mockery::mock( Options::class );
-		$this->homepage_preloader = Mockery::mock( Homepage::class );
-		$this->subscriber  = new Subscriber( $this->settings, $this->database, $this->usedCSS, $this->options_api, $this->homepage_preloader );
+		$this->settings   = Mockery::mock( Settings::class );
+		$this->database   = Mockery::mock( Database::class );
+		$this->usedCSS    = Mockery::mock( UsedCSS::class );
+		$this->queue      = new Queue();
```

Comment:
Why not use a mocked object?

Reply:
Really I don't know the reason behind the following error when trying to mock the Query class
```

Use of undefined constant OBJECT - assumed 'OBJECT' (this will throw an Error in a future version of PHP)
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/vendor/mockery/mockery/library/Mockery/Generator/Parameter.php:40
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/vendor/mockery/mockery/library/Mockery/Generator/StringManipulation/Pass/MethodDefinitionPass.php:78
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/vendor/mockery/mockery/library/Mockery/Generator/StringManipulation/Pass/MethodDefinitionPass.php:47
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/vendor/mockery/mockery/library/Mockery/Generator/StringManipulationGenerator.php:79
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/vendor/mockery/mockery/library/Mockery/Generator/CachingGenerator.php:40
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/vendor/mockery/mockery/library/Mockery/Container.php:213
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/vendor/mockery/mockery/library/Mockery.php:123
 /var/www/html/wordpress/wp-content/plugins/wp-rocket/tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Subscriber/cleanUsedCssAndCache.php:38

```

Reply:
It's a constant that is created by WP in the DB class, so Mockery don't know about it by default. I added it to the bootstrap file and updated the tests